model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -83,18 +83,129 @@ elif [[ "$OSTYPE" == "darwin"* ]]; then
     python3 -m pip install --upgrade pip
 fi
 
+install_latest_ov() {
+    if $2; then
+        sudo -E "$1" -m pip install openvino
+    else
+        "$1" -m pip install openvino
+    fi
+}
+
+install_ov() {
+    if $2; then
+        sudo -E "$1" -m pip install openvino=="$3"
+    else
+        "$1" -m pip install openvino=="$3"
+    fi
+}
+
+uninstall_ov() {
+    if $2; then
+        sudo -E "$1" -m pip uninstall -y openvino
+    else
+        "$1" -m pip uninstall -y openvino
+    fi
+}
+
+check_ie() {
+    $1 "$SCRIPTDIR/../mo/utils/find_ie_version.py"
+}
+
+check_ov_package() {
+    if $2; then
+        sudo -E "$1" -m pip show openvino
+    else
+        "$1" -m pip show openvino
+    fi
+}
+
+print_warning() {
+    YELLOW='\033[1;33m'
+    NC='\033[0m'
+    printf "${YELLOW}[ WARNING ] %s ${NC}\n" "$1"
+}
+
+find_ie_bindings() {
+    python_executable="$1"
+    requires_sudo="$2"
+
+    mo_release_version="$("$python_executable" "$SCRIPTDIR"/../mo/utils/extract_release_version.py)"
+    if [[ $mo_release_version == "None.None" ]]; then
+      mo_is_custom=true
+    else
+      mo_is_custom=false
+    fi
+
+    if ! check_ie "$python_executable"; then
+        # check pip installed openvino package
+        if check_ov_package "$python_executable" "$requires_sudo"; then
+            if $mo_is_custom; then
+                print_warning "Existing \"OpenVINO (TM) Toolkit\" pip version is incompatible with Model Optimizer."
```

Comment:
```suggestion
                print_warning "Pip version in the existing \"OpenVINO (TM) Toolkit\" is incompatible with the Model Optimizer."
```

Reply:
No :) we don't compare pip version with MO.
And for me the initial message is clear that existing version of OpenVINO installed in pip is incompatible...
"OpenVINO (TM) Toolkit version installed in pip is incompatible with the Model Optimizer." - is it ok?

Reply:
Done.